source/Marquee.js

```diff
@@ -780,6 +794,7 @@
 		* @private
 		*/
 		_marquee_contentChanged: function () {
+			this._marquee_setTextOverflow();
```

Comment:
Hi Anish,
I think we need to test length after setting content on marquee text. How do you think?